ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m

```diff
@@ -0,0 +1,38 @@
+//
+// Created by RevenueCat.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+#import "RCPurchases+HybridAdditions.h"
+#import <Purchases/Purchases.h>
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@implementation RCPurchases (HybridAdditions)
+
++ (instancetype)configureWithAPIKey:(NSString *)APIKey
+                          appUserID:(nullable NSString *)appUserID
+                       observerMode:(BOOL)observerMode
+              userDefaultsSuiteName:(nullable NSString *)userDefaultsSuiteName
```

Comment:
How are we going to surface this in the hybrid SDKs? What if we always instantiate using a special name instead? Like what if we do:
`[[NSUserDefaults alloc] initWithSuiteName:@"revenuecat"];`

Would that fix the issues?

Reply:
that's a good idea. my one concern though is potential issues with migration - if there are unsynced subscriber attributes and such in the old user defaults, they'd get lost

Reply:
ohhh good thing you thought about that. Let's leave it this way then 👍